docs/src/examples/xxz_dmrg.md

```diff
@@ -12,14 +12,11 @@ Instead of constructing explicit matrices from Kronecker products as we did befo
 For that purpose, we first import
 
 ```@example xxz_dmrg; continued = true
-using LinearAlgebra
-using ITensors
-using Plots
-using ReducedBasis
+using LinearAlgebra, ITensors, Plots, ReducedBasis
```

Comment:
Not too big a fan of putting these on one line, acutally. It makes it easy to miss one thing.

Reply:
I thought it was okay for the examples, since they have not that many dependencies, but I can change it back to the initial formatting.